interface/modules/zend_modules/module/Patientvalidation/src/Patientvalidation/Controller/PatientvalidationController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace Patientvalidation\Controller;
+
+use Zend\Mvc\Controller\AbstractActionController;
+use Zend\View\Model\ViewModel;
+
+class PatientvalidationController extends BaseController{
+
+    /**
+     * PatientvalidationController constructor.
+     */
+    public function __construct()
+    {
+        //todo add permission of admin
+
+    }
+
+
+    /**
+     * @return \Zend\Stdlib\ResponseInterface the index action
+     */
+    public function indexAction()
+    {
+        //here we can conduct all the validation we need , on the patient we need to check
+       return  $this->responseWithNoLayout(array("status"=>"ok"));
```

Comment:
Would be good to place a couple example here of what can be done (and comment them out).